code/__defines/time.dm

```diff
@@ -83,9 +83,13 @@ GLOBAL_LIST_INIT(month_seconds_cumulative_leap, list(
 ))
 
 
+#define TICKS *world.tick_lag
```

Comment:
these three defines need explanation, what are they ? what do they do?

Reply:
It's doing the same thing as #define SECONDS. Just to make code more readable. I probably need to replace some *world.tick_lag with a define

Reply:
TICKS doing the same thing as SECONDS;

DS2TICKS mean "delay seconds to ticks". It was originally used in /proc/stoplag() (used instead of sleep(1) or sleep() ) but we don't have it and I'm not going to port it (at least not in this PR);

TICKS2DS mean "ticks to delay seconds).